Simpson_Assign1/Program.cs

```diff
@@ -16,11 +16,131 @@ static void Main(string[] args)
             Students = InitializeStudents();
             Courses = InitializeCourses();
 
+            string input = "";
             do
             {
+                Console.WriteLine("\nPlease choose from the following options:\n");
                 PrintMenu();
+                input = Console.ReadLine();
+
+                int count = 0;
```

Comment:
The way that you are using this count variable isn't necessary because a List already has a count property that you could use in your if statements instead.